gitlab/unit-tests.sh

```diff
@@ -2,6 +2,16 @@
 
 set -euxo pipefail
 
+add-apt-repository ppa:ondrej/php -y
+apt-get update
+
+PACKAGES=$(dpkg-query -f '${binary:Package}\n' -W)
```

Comment:
Shouldn't we check here that we only do this for php packages? i.e. aren't we now also uninstalling for example `curl` or `nginx`?

Reply:
Yes, I already fixed that, also I'm not sure if for example php8.0-pear will actually exist so I'm still investigating if this does work.